### PR TITLE
Mooc2022 supportmatrix

### PR DESCRIPTION
**Pull request by frank-qcd-qk**
In reference to task: [DTSW-1206](REDACTED) 

[DTSW-1206]: https://ethidsc.atlassian.net/browse/DTSW-1206?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by frank-qcd-qk** [comment]
Pending to perform actual robot test

**Comment by frank-qcd-qk** [comment]
image

File: README.md [hunk 1]
```diff
@@ -144,6 +144,18 @@ in the notebooks to see if you need to access VNC.
 This simulation test is just that, a test. Don't trust it fully. If you want a more accurate
 metric of performance, continue reading to the `Perform local evaluation` section below.
 
+### ℹ️ Check Robot Compatibility
+While we try our best to support running these exercises on all Duckiebot, some activities require special hardware and
```
Comment (kghite) [inline README.md:148]: "all versions of the Duckiebot"

File: braitenberg/README.md [hunk 2]
```diff
@@ -14,6 +14,19 @@ Your submissions will be sent to the [`lx22-BV1`][challenge] challenge.
 
 [challenge]: https://challenges.duckietown.org/v4/humans/challenges/lx22-BV1
 
+## ℹ️️ Check Robot Compatibility
```
Comment (kghite) [inline braitenberg/README.md:17]: I would consider adding this as a subsection to the Testing on a Physical Robot portion of each exercise README to standardize where it appears in the instructions.

Comment (frank-qcd-qk) [reply braitenberg/README.md:17]: Can you elaborate a bit more? do you want to name it "Testing on a Physical Robot"? @afdaniele can you weigh in?

**Comment by frank-qcd-qk** [comment]
Updated: 
image

File: collision-checker/README.md [hunk 3]
```diff
@@ -76,6 +76,33 @@ Inside the code editor, use the navigator sidebar on the left-hand side to navig
...
+    dts code workbench --duckiebot YOUR_DUCKIEBOT
+
+This is the modality "everything runs on the robot".
+
+You can also test using
+
+    dts code workbench --duckiebot YOUR_DUCKIEBOT --local 
+
+This is the modality "drivers running on the robot, agent runs on the laptop."
+
```
Comment (afdaniele) [inline collision-checker/README.md:105]: The exercise `collision-checker` does not run on the physical robot, you can remove this.

File: planning/README.md [hunk 4]
```diff
@@ -26,6 +26,20 @@ You can verify the scores of your submissions on the [Challenges website][challe
...
+robot.
+
+The support matrix of this exercise is as follows:
+
+| Duckiebot Type   	                                                                                | Configuration 	 | Support Level   	    |
+|---------------------------------------------------------------------------------------------------|-----------------|----------------------|
+| [DB21-J4](https://get.duckietown.com/products/duckiebot-db21?variant=41543707099311)            	 | Jetson 4GB    	 | ✔️ Full Support    	 |
+| [DB21-J2](https://get.duckietown.com/products/duckiebot-db21?variant=40700056830127)            	 | Jetson 2GB    	 | ✔️ Full Support 	    |
+
+
```
Comment (afdaniele) [inline planning/README.md:42]: The exercise `planning` does not run on the physical robot, you can remove this.

File: planning/README.md [hunk 5]
```diff
@@ -73,6 +87,33 @@ Inside the code editor, use the navigator sidebar on the left-hand side to navig
...
+    dts code workbench --duckiebot YOUR_DUCKIEBOT
+
+This is the modality "everything runs on the robot".
+
+You can also test using
+
+    dts code workbench --duckiebot YOUR_DUCKIEBOT --local 
+
+This is the modality "drivers running on the robot, agent runs on the laptop."
+
```
Comment (afdaniele) [inline planning/README.md:116]: Again, the exercise `planning` does not run on the physical robot, you can remove this.

**Review by afdaniele — CHANGES_REQUESTED** [review]

**Comment by frank-qcd-qk** [comment]
@afdaniele @kghite I will let you guys merge into mooc2022 branch...

**Pull request merged**